ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -1,273 +0,0 @@
-base_images:
-  ansible:
-    name: "4.11"
-    namespace: ocp
-    tag: ansible
-  aws-efs-csi-operator-create-efs:
-    name: "4.11"
-    namespace: ocp
-    tag: aws-efs-csi-operator-create-efs
-  cli:
-    name: "4.11"
-    namespace: ocp
-    tag: cli
-  dev-scripts:
-    name: test
-    namespace: ocp-kni
-    tag: dev-scripts
-  openstack-installer:
-    name: "4.11"
-    namespace: ocp
-    tag: openstack-installer
-  tests-private-postupg:
-    name: tests-private
-    namespace: ci
-    tag: "4.12"
-  tests-private-preupg:
-    name: tests-private
-    namespace: ci
-    tag: "4.11"
-  tools:
-    name: "4.11"
-    namespace: ocp
-    tag: tools
-  upi-installer:
-    name: "4.11"
-    namespace: ocp
-    tag: upi-installer
-  verification-tests:
-    name: verification-tests
-    namespace: ci
-    tag: latest
-build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.18
-releases:
-  arm64-latest:
-    prerelease:
-      architecture: arm64
-      product: ocp
-      version_bounds:
-        lower: 4.11.0-0
-        upper: 4.12.0-0
-  arm64-target:
-    candidate:
-      architecture: arm64
-      product: ocp
-      stream: nightly
-      version: "4.12"
-  latest:
-    prerelease:
-      architecture: amd64
-      product: ocp
-      version_bounds:
-        lower: 4.11.0-0
-        upper: 4.12.0-0
-  target:
-    candidate:
-      architecture: amd64
-      product: ocp
-      stream: nightly
-      version: "4.12"
-resources:
-  '*':
-    requests:
-      cpu: 100m
-      memory: 200Mi
-tests:
-- as: aws-ipi-byo-iam-role-f60
```

Comment:
To be consistent with other EOL versions(such as [4.10-4.11 arm job](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10.yaml))share we keep the yaml file with updating the frequency from f60 to f360?

Reply:
I am ok with f360 for all one-hop upgrade and will leave CPOU upgrade frequency as-is.

Reply:
> I am ok with f360 for all one-hop upgrade
All 4.12 arm64 upgrade yaml files are still deleted, could u double check?

> will leave CPOU upgrade frequency as-is
Can you share any special case for this one? maybe we should keep the same rule on it for others to follow up.

Reply:
> All 4.12 arm64 upgrade yaml files are still deleted, could u double check?

as the MR description said `ocp 4.12 removed the support of arm architecture, so we don't need to test them in upgrade ci anymore`

Reply:
> Can you share any special case for this one? 

All jobs in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-cpou-upgrade-from-4.10.yaml

Reply:
> To be consistent with other EOL versions(such as [4.10-4.11 arm job](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10.yaml))share we keep the yaml file with updating the frequency from f60 to f360?

I meant, for the those profiles which won't to be tested in 4.12 arm upgrade yamls, shall we keep the yaml file instead of deleting them, we could just adjust the test frequency from f60 to f360 to drop the test. This seems a general rule. Referring to 4.10-4.11 arm jobs, even we don't test 4.11 arm upgrade any more, but we still keep the yaml file while test frequency is f360.

Reply:
ok, will add them back

Reply:
![Screenshot from 2024-10-15 11-01-43](https://github.com/user-attachments/assets/34ba87a9-d555-4834-9334-f994446a6083)

All 4.12 arm upgrade yaml files(Whether it's cpou or one hop) are deleted directly in your pr ^^

Reply:
only the listed 3 config files

Reply:
done